sui/tests/shared_objects_tests.rs

```diff
@@ -66,8 +66,14 @@ async fn submit_shared_object_transaction(
 
         let mut replies = Vec::new();
         for result in futures::future::join_all(futures).await {
-            replies.push(Some(result))
+            let reply = match &result {
+                Ok(_) => Some(result),
+                Err(SuiError::ConsensusConnectionBroken(..)) => None,
+                Err(..) => Some(result),
+            };
+            replies.push(reply)
         }
+
         if replies.iter().any(|x| x.is_some()) {
             // Remove all `ConsensusConnectionBroken` replies.
```

Comment:
This comment seems to indicate the filtering out of those errors happens below?

Reply:
no it doesn't, but your question indicates that I should write this code better :p let me have another try

Reply:
fixed